demo/index.html

```diff
@@ -16,6 +16,10 @@
     <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
 
     <link rel="import" href="../../polymer/polymer.html">
+
+    <link rel="import" href="../../paper-styles/paper-styles.html">
+    <link rel="import" href="../../paper-styles/demo-pages.html">
```

Comment:
If you're not using the `.vertical-section` and friends, you don't really need to include `demo-pages` and `paper-styles`. The roboto font can come from `demo.css alone`, and you can just style the body.